diff --git a/node_modules/@ledgerhq/hw-app-eth/.turbo/turbo-build.log b/node_modules/@ledgerhq/hw-app-eth/.turbo/turbo-build.log
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/CHANGELOG.md b/node_modules/@ledgerhq/hw-app-eth/CHANGELOG.md
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/README.md b/node_modules/@ledgerhq/hw-app-eth/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/erc20.js b/node_modules/@ledgerhq/hw-app-eth/erc20.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/jest.config.ts b/node_modules/@ledgerhq/hw-app-eth/jest.config.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/errors.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/EIP712.utils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/modules/EIP712/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/contracts.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/erc20.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/loadConfig.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/ledger/nfts.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/services/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.js.map b/node_modules/@ledgerhq/hw-app-eth/lib-es/utils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/errors.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/errors.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/errors.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/errors.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/errors.js b/node_modules/@ledgerhq/hw-app-eth/lib/errors.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/errors.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/errors.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.js b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.js b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.js
old mode 100644
new mode 100755
index 2aa2d39..1da9703
--- a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.js
+++ b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.js
@@ -68,7 +68,7 @@ exports.getFiltersForMessage = exports.getSchemaHashForMessage = exports.sortObj
 var axios_1 = __importDefault(require("axios"));
 var sha224_1 = __importDefault(require("crypto-js/sha224"));
 var utils_1 = require("../../utils");
-var eip712_1 = __importDefault(require("@ledgerhq/cryptoassets/data/eip712"));
+var eip712_1 = __importDefault(require("@ledgerhq/cryptoassets/lib/data/eip712"));
 var bignumber_js_1 = __importDefault(require("bignumber.js"));
 var NULL_ADDRESS = "0x0000000000000000000000000000000000000000";
 /**
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/EIP712.utils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.js b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/modules/EIP712/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.js b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/contracts.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.js b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.js
old mode 100644
new mode 100755
index 5d1643b..35eb8f2
--- a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.js
+++ b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.js
@@ -42,7 +42,7 @@ exports.__esModule = true;
 exports.byContractAddressAndChainId = exports.findERC20SignaturesInfo = void 0;
 var axios_1 = __importDefault(require("axios"));
 var logs_1 = require("@ledgerhq/logs");
-var index_1 = require("@ledgerhq/cryptoassets/data/evm/index");
+var index_1 = require("@ledgerhq/cryptoassets/lib/data/evm/index");
 var loadConfig_1 = require("./loadConfig");
 var asContractAddress = function (addr) {
     var a = addr.toLowerCase();
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/erc20.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.js b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.js b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/loadConfig.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.js b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/ledger/nfts.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/types.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/services/types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/types.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/types.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/types.js b/node_modules/@ledgerhq/hw-app-eth/lib/services/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/services/types.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/services/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/utils.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/utils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/utils.d.ts.map b/node_modules/@ledgerhq/hw-app-eth/lib/utils.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/utils.js b/node_modules/@ledgerhq/hw-app-eth/lib/utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/utils.js.map b/node_modules/@ledgerhq/hw-app-eth/lib/utils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/Eth.ts b/node_modules/@ledgerhq/hw-app-eth/src/Eth.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/errors.ts b/node_modules/@ledgerhq/hw-app-eth/src/errors.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/modules/EIP712/EIP712.types.ts b/node_modules/@ledgerhq/hw-app-eth/src/modules/EIP712/EIP712.types.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/modules/EIP712/EIP712.utils.ts b/node_modules/@ledgerhq/hw-app-eth/src/modules/EIP712/EIP712.utils.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/modules/EIP712/index.ts b/node_modules/@ledgerhq/hw-app-eth/src/modules/EIP712/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/contracts.ts b/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/contracts.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/erc20.ts b/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/erc20.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/index.ts b/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/loadConfig.ts b/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/loadConfig.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/nfts.ts b/node_modules/@ledgerhq/hw-app-eth/src/services/ledger/nfts.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/services/types.ts b/node_modules/@ledgerhq/hw-app-eth/src/services/types.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/utils.ts b/node_modules/@ledgerhq/hw-app-eth/src/utils.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/EIP712/filtered-signMessage.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/EIP712/filtered-signMessage.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/EIP712/noFilter-signMessage.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/EIP712/noFilter-signMessage.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/EIP712/utils.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/EIP712/utils.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/ERC20/ERC20-CAL-KO.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/ERC20/ERC20-CAL-KO.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/ERC20/ERC20-CAL-OK.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/ERC20/ERC20-CAL-OK.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/Eth.integration.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/Eth.integration.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/Eth.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/Eth.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/ERC1155.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/ERC1155.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/ERC20.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/ERC20.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/ERC721.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/ERC721.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/PARASWAP.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/ABI/PARASWAP.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/CAL.ts b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/CAL.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/REST/Paraswap-Plugin.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/REST/Paraswap-Plugin.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/SignatureCALEth.ts b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/SignatureCALEth.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/0-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/0-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/0.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/0.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/1-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/1-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/1.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/1.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/10-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/10-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/10.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/10.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/11-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/11-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/11.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/11.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/12-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/12-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/12.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/12.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/13-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/13-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/13.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/13.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/2-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/2-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/2.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/2.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/3-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/3-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/3.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/3.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/4-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/4-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/4.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/4.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/5-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/5-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/5.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/5.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/6-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/6-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/6.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/6.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/7-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/7-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/7.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/7.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/8-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/8-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/8.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/8.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/9-filtered.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/9-filtered.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/9.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/9.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/ERC20-KO.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/ERC20-KO.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/ERC20-OK.apdus b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/apdus/ERC20-OK.apdus
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/dynamicCAL.ts b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/dynamicCAL.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/0.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/0.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/1.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/1.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/10.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/10.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/11.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/11.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/12.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/12.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/13.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/13.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/2.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/2.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/3.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/3.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/4.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/4.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/5.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/5.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/6.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/6.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/7.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/7.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/8.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/8.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/9.json b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/messages/9.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/utils.ts b/node_modules/@ledgerhq/hw-app-eth/tests/fixtures/utils.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/ledgerService.unit.test.ts b/node_modules/@ledgerhq/hw-app-eth/tests/ledgerService.unit.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tests/paraswap.json b/node_modules/@ledgerhq/hw-app-eth/tests/paraswap.json
old mode 100644
new mode 100755
diff --git a/node_modules/@ledgerhq/hw-app-eth/tsconfig.json b/node_modules/@ledgerhq/hw-app-eth/tsconfig.json
old mode 100644
new mode 100755
