diff --git a/node_modules/react-native-keychain/LICENSE b/node_modules/react-native-keychain/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/README.md b/node_modules/react-native-keychain/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/RNKeychain.podspec b/node_modules/react-native-keychain/RNKeychain.podspec
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/RNKeychain.xcodeproj/project.pbxproj b/node_modules/react-native-keychain/RNKeychain.xcodeproj/project.pbxproj
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.h b/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.h
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m b/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/README.md b/node_modules/react-native-keychain/android/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/AndroidManifest.xml b/node_modules/react-native-keychain/android/src/main/AndroidManifest.xml
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/DeviceAvailability.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/DeviceAvailability.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/KeychainModule.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/KeychainModule.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/KeychainModuleBuilder.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/KeychainModuleBuilder.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/KeychainPackage.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/KeychainPackage.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/PrefsStorage.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/PrefsStorage.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/SecurityLevel.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/SecurityLevel.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorage.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorage.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageFacebookConceal.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageFacebookConceal.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreAesCbc.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreAesCbc.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreRsaEcb.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreRsaEcb.java
old mode 100644
new mode 100755
index b191bdc..94ecaa7
--- a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreRsaEcb.java
+++ b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreRsaEcb.java
@@ -125,18 +125,8 @@ public class CipherStorageKeystoreRsaEcb extends CipherStorageBase {
       // key is always NOT NULL otherwise GeneralSecurityException raised
       key = extractGeneratedKey(safeAlias, level, retries);
 
-      final DecryptionResult results = new DecryptionResult(
-        decryptBytes(key, username),
-        decryptBytes(key, password)
-      );
-
-      handler.onDecrypt(results, null);
-    } catch (final UserNotAuthenticatedException ex) {
-      Log.d(LOG_TAG, "Unlock of keystore is needed. Error: " + ex.getMessage(), ex);
-
-      // expected that KEY instance is extracted and we caught exception on decryptBytes operation
       @SuppressWarnings("ConstantConditions") final DecryptionContext context =
-        new DecryptionContext(safeAlias, key, password, username);
+      new DecryptionContext(safeAlias, key, password, username);
 
       handler.askAccessPermissions(context);
     } catch (final Throwable fail) {
@@ -238,7 +228,7 @@ public class CipherStorageKeystoreRsaEcb extends CipherStorageBase {
       .setEncryptionPaddings(PADDING_PKCS1)
       .setRandomizedEncryptionRequired(true)
       .setUserAuthenticationRequired(true)
-      .setUserAuthenticationValidityDurationSeconds(5)
+      .setUserAuthenticationParameters(5, KeyProperties.AUTH_BIOMETRIC_STRONG)
       .setKeySize(keySize);
   }

diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandler.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandler.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerInteractiveBiometric.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerInteractiveBiometric.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerInteractiveBiometricManualRetry.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerInteractiveBiometricManualRetry.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerNonInteractive.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerNonInteractive.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerProvider.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/decryptionHandler/DecryptionResultHandlerProvider.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/exceptions/CryptoFailedException.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/exceptions/CryptoFailedException.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/exceptions/EmptyParameterException.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/exceptions/EmptyParameterException.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/exceptions/KeyStoreAccessException.java b/node_modules/react-native-keychain/android/src/main/java/com/oblador/keychain/exceptions/KeyStoreAccessException.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/index.js b/node_modules/react-native-keychain/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-keychain/typings/react-native-keychain.d.ts b/node_modules/react-native-keychain/typings/react-native-keychain.d.ts
old mode 100644
new mode 100755
